crowbar_framework/lib/crowbar/deployment_queue.rb

```diff
@@ -130,29 +129,29 @@ def process_queue
         begin
           f = file_lock.acquire("queue", logger: logger)
 
-          if proposal_queue.empty?
+          if @proposal_queue.nil?
             logger.debug("process queue: exit: empty queue")
             return
           end
 
-          logger.debug("process queue: queue: #{proposal_queue.proposals.inspect}")
+          logger.debug("process queue: queue: #{@proposal_queue.inspect}")
 
           # Test for ready
           remove_list = []
-          proposal_queue.proposals.each do |item|
-            prop = Proposal.where(barclamp: item["barclamp"], name: item["inst"]).first
+          ProposalQueue.all.each do |item|
+            prop = Proposal.where(barclamp: item.barclamp, name: item.name).first
 
             if prop.nil?
-              remove_list << item
+              remove_list << { barclamp: item.barclamp, inst: item.name }
               next
             end
 
-            next unless dependencies_satisfied?(item["deps"])
+            next unless dependencies_satisfied?(item.properties["deps"])
 
-            nodes_map = elements_to_nodes_to_roles_map(prop["deployment"][item["barclamp"]]["elements"],
-                                                       prop["deployment"][item["barclamp"]]["element_order"])
+            nodes_map = elements_to_nodes_to_roles_map(prop["deployment"][item.barclamp]["elements"],
+                                                       prop["deployment"][item.barclamp]["element_order"])
```

Comment:
Style/AlignParameters: Align the parameters of a method call if they span more than one line.